content/en/docs/reference/glossary/qos-class.md

```diff
@@ -4,7 +4,7 @@ id: qosClass
 date: 2019-04-15
 full_link: 
 short_description: >
-  QoSClass provides a way for Kubernetes to classify pods within the cluster into several classes and make decisions about scheduling and eviction.
+  QoSClass (Quality of Service Class) provides a way for Kubernetes to classify pods within the cluster into several classes and make decisions about scheduling and eviction.
 
```

Comment:
hi, @nelvadas.
This page may help clarify the name: https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/#create-a-pod-that-gets-assigned-a-qos-class-of-guaranteed.
Quality of Service (QoS) class seems to make sense.

Reply:
@nelvadas  can you please address the suggestions made by @kbhawkey .